.github/workflows/ci.yml

```diff
@@ -41,3 +41,14 @@ jobs:
         image: memcached:latest
         ports:
         - 11211/tcp
+      tls_memcached:
+        image: memcached:latest
+        ports:
+        - 11212:11211/tcp
+        volumes:
+        - ./extras/certs/gen:/opt/certs
+        command: >-
```

Comment:
I think you want `options:` instead of `command:`?

Reply:
I tried with options, it passes them as options to the docker run command, not to the container itself. I'm creating a convenience docker image to have all the needed bits inside as I'm doing this over and over across several projects.